tests/unit/commands/_utils/test_options.py

```diff
@@ -527,3 +528,20 @@ def test_skip_without_hook_package(self):
             skip_prepare_infra_callback(ctx_mock, param_mock, True)
 
         self.assertEqual(str(ex.exception), "Missing option --hook-name")
+
+
+class TestNextCommandSuggestions(TestCase):
+    def test_generate_next_command_recommendation(self):
+        listOfTuples = [("command1", "description1"), ("command2", "description2"), ("command3", "description3")]
+        output = generate_next_command_recommendation(listOfTuples)
+        expectedOutput = """
+Commands you can use next:
+================================
+
+[*] command1: description1
+[*] command2: description2
```

Comment:
Why change how it currently looks?

Currently if I run `sam init`, I get:

```
    Commands you can use next
    =========================
    [*] Create pipeline: cd sam-app && sam pipeline init --bootstrap
    [*] Validate SAM template: cd sam-app && sam validate
    [*] Test Function in the Cloud: cd sam-app && sam sync --stack-name {stack-name} --watch
```

The underline isn't aligned to the text width, there's an extra newline, and it's not clear to me what value the final `================================` on L545 adds.

Reply:
I'll make it so that it mimics the old appearance